schemas/extractor.yml

```
@@ -27,6 +27,44 @@ classes:
                     to this file type.
 
 
+    Usage:
+        attributes:
+            method:
+                required: true
+                range: UsageTypes
+                description: >-
+                    Usage invocation method, e.g. from a command line or from Python.
+            command:
+                required: true
+                description: >-
+                    A machine-executable command by which the Extractor functionality
+                    can be accessed. For method:cli, this is a parametrized executable
+                    invocation which should be available after installing the Extractor.
+                    For method:python, this is a parametrized function invocation
+                    which should be available after installing and importing the
+                    Extractor.
+
+
+    Installation:
+        attributes:
+            method:
+                required: true
+                range: InstallationTypes
+                description: >-
+                    Extractor installation method, specifying the package manager
+                    to
+                    process the installation instructions with.
+            requirements:
+                description: >-
+                    The contents of an optional 'requirements.yml'-like file, which
```

```suggestion
                    The contents of an optional 'requirements.txt'-like file, which
```

is this a new thing or should it be .txt?